src/DataGrid.tsx

```diff
@@ -1185,6 +1195,9 @@ export function DataGrid<R, SR = unknown, K extends Key = Key>(props: DataGridPr
       aria-multiselectable={isSelectable ? true : undefined}
       aria-colcount={columns.length}
       aria-rowcount={ariaRowCount}
+      // Scrollable containers without tabIndex are keyboard focusable in Chrome only if there is no focusable element inside
+      // whereas they are always focusable in Firefox. We need to set tabIndex to have a consistent behavior across browsers.
```

Comment:
Should we do that?
Maybe Firefox users want to preserve the default behavior, wdyt?

Reply:
Not sure if it is helpful. It is good to have consistent behavior across browsers though. I checked a few libraries and they all work similarly where the first cell receives focus. Also shift + tab does not focus the grid 🤔 in FF. I think we can merge this and revisit if there any issues.